services/app-provider/pkg/config/config.go

```diff
@@ -26,6 +27,11 @@ type Config struct {
 	Context    context.Context `yaml:"-"`
 }
 
+type Registry struct {
+	Type      string   `yaml:"type" env:"MICRO_REGISTRY"`
+	Addresses []string `yaml:"addresses" env:"MICRO_REGISTRY_ADDRESSES"`
```

Comment:
Should't those envs start with `OICS_` or `APP_PROVIDER_` ?

Reply:
Maybe to _add_ new ones in parallel having the correct name as we "later" plan a deprication process anyways...?

Reply:
Actually in this case it might even need to stay `MICRO_REGISTRY` (haven't checked in detail) as go-micro itself make use of that variable.

Reply:
#3917 (Harmonize env variable naming for those which are not bound to a particular service)